test/serializer/abstract/QualifiedCall.js

```diff
@@ -0,0 +1,24 @@
+// add at runtime: global.bar = {x: 1};
+let bar = global.__abstract ? __makeSimple(__abstract('object', 'bar')) : {x: 1};
+
+let foo = global.__abstract ? __abstract('function', '(function() { return this.x; })') : function() { return this.x; };
+bar.foo = foo;
+x = bar.foo();
+
+let foo2 = global.__abstract ? __abstract('function', '(function(a) { return this.x + a; })') : function(a) { return this.x + a; };
+bar[1] = foo2;
+y = bar[1](100);
+
+// Uncomment this when assignments to symbol properties are supported
```

Comment:
This is not supported by the spec, is it? Symbols behave like other value types. Meaning they get coerced into an object temporarily but the object is dropped.

Reply:
I don't think I understand your point. This works in Node:

$ node
> let fooSym = Symbol.for("foo")
undefined
> fooSym
Symbol(foo)
> let bar = {}
undefined
> bar[fooSym] = function() { return this; }
[Function]
> bar[fooSym]()
{ [Symbol(foo)]: [Function] }
> bar[fooSym] = function(x) { return x + this.y; }
[Function]
> bar.y = 2;
2
> bar[fooSym](3)
5
>

Reply:
Oh never mind my comment. I read this as `fooSym[bar]`. Carry on.